src/gtk/toga_gtk/app.py

```diff
@@ -107,6 +107,16 @@ def gtk_startup(self, data=None):
         settings = Gtk.Settings.get_default()
         settings.set_property("gtk-shell-shows-menubar", False)
 
+        # Set any custom styles
+        css_provider = Gtk.CssProvider()
+        css_provider.load_from_data(STYLES)
```

Comment:
I take it this is an "application wide" style sheet? If so, we may want to rename `STYLES` to reflect that.

Reply:
Yes, it is application wide. What do you think would be a good name?
Note: At the moment it only has one class that does not overwrite anything native to gtk.